### DISCUSSION TITLE
Installation of KnowWE on a linux server

### DISCUSSION THREAD

**Discussion by VitaliyMez** [denkbares/d3web-KnowWE#12] [Q&A] [answered]
Hi, is there a possibility to install KnowWE on ubuntu?
Thank you!

**Comment by AlbrechtStriffler** [comment, marked as answer]
Hi, yes - as most Java applications, Tomcat and KnowWE run natively on Ubuntu and other Linux distros. We basically host all our KnowWE instances on Linux servers. See https://www.d3web.de/Wiki.jsp?page=D3web-KnowWE%20Quickstart for more infos (check file persmissions and so on).

**Reply by AlbrechtStriffler** [reply]
The quickstart guide is a bit outdated, but basically the only relevant part for linux is the following:
![image](https://user-images.githubusercontent.com/13681360/166491905-b125485f-a390-47e8-be6e-70f8e99b0681.png)
